charts/dapr/charts/dapr_scheduler/templates/dapr_scheduler_poddisruptionbudget.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if and (eq .Values.global.ha.enabled true) (eq .Values.global.scheduler.enabled true) }}
```

Comment:
Do you also need to duplicate the HA vs replica count here too? Maybe just rely on replica count?

Reply:
I thought global ha impacted all control plane services, but I see now it doesn't impact placement and I guess we can enforce it to not impact scheduler. Thats fine. I'll rm the global ha and set it to just the scheduler `replicaCount`

Reply:
resolved